Providers/Profile.swift

```diff
@@ -958,21 +958,16 @@ public class BrowserProfile: Profile {
         private func displayStateForSyncState(syncStatus: SyncStatus, identifier: String? = nil) -> SyncDisplayState {
             switch syncStatus {
             case .Completed:
-                return SyncDisplayState.Good
-            case .NotStarted(let reason):
```

Comment:
By removing the .NotStarted case wont the browser just default to Good even when the sync has failed. You mentioned that the SyncOfflineTitle was wrong. But in the case of the sync not starting for various reasons the NotStartedTitle would still be helpful instead of making them think sync succeeded.